pyproject.toml

```diff
@@ -11,6 +11,7 @@ dependencies = [
     "botocore==1.35.96",
     "distro==1.9.0",
     "django==4.2.16",
+    "django-ansible-base>=2025.1.3",
```

Comment:
will this go to ansible.cfg used for prod dependencies?

Assuming this dependency won't be present in 2.4, is there a way to do optional dependency?

Reply:
seems like there is extras_require https://stackoverflow.com/questions/10572603/specifying-optional-dependencies-in-pypi-python-setup-py 

so we should be able to use that in prod mode, so it fetches dab only when it's there?

Reply:
My understanding is that the RPM build would only use setup.cfg, but not pyproject.toml.

If it uses both, we'd probably need to move the standalone util into a separate repo..

Reply:
OK, update from Satoe .. the RPM builds *might* use `pyproject.toml` first if both exist, but it is possible to explicitly say to use `setup.cfg` only for these.

So.. probably something to keep in mind when doing a release, but .. ok for now?